app/src/components/common/TmConnectedNetwork.vue

```diff
@@ -36,11 +36,19 @@
         <router-link
           v-if="block.height"
           v-tooltip.top="'Block Height'"
-          class="block-number"
-          :to="{
-            name: `block`,
-            params: { height: block.height, networkId: networkSlug },
-          }"
+          :class="
+            `block-number` && {
+              noClickable: currentNetwork.network_type === `polkadot`,
```

Comment:
why don't we use the feature flag instead of making it network specific?

Reply:
we can do that as well. But then it is clickable and will take you to the "Feature not available" page

Reply:
whichever you prefer

Reply:
```suggestion
              noClickable: !currentNetwork.feature_activity,
```